index.js

```diff
@@ -19,4 +19,150 @@ btnClose.addEventListener('click', closeMobileNav);
 menuBtn.addEventListener('click', navToggle);
 menuLinks.forEach((navLink) => {
   navLink.addEventListener('click', closeMobileNav);
+});
+
+// Project Data
+const projectData = [
+  {
+    image: './img/tonic-01.png',
```

Comment:
- [ ] **Optional:** The images are blurred on the Desktop view. Kindly consider reducing the size of the popup container and so the width of the image.

Reply:
Hello @brytebee, thank you so much for your recommendations, I will apply them.